jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/FilesTaskV2Test.java

```diff
@@ -52,8 +52,12 @@ private static String verifyTaskSuccess(TestProject project, @Nullable String mo
     BuildTask jibTask = buildResult.task(taskName);
     Assert.assertNotNull(jibTask);
     Assert.assertEquals(TaskOutcome.SUCCESS, jibTask.getOutcome());
+    String output = buildResult.getOutput().trim();
+    Assert.assertThat(output, CoreMatchers.startsWith("BEGIN JIB JSON"));
+    Assert.assertThat(output, CoreMatchers.endsWith("END JIB JSON"));
 
-    return buildResult.getOutput().trim();
+    // Return task output with header/footer removed
+    return output.replace("BEGIN JIB JSON", "").replace("END JIB JSON", "").trim();
```

Comment:
nit: The `@return` should probably mention the JSON rather than just a list of paths.